akirachix/Upile-Informational
# added navbar and the landing page for lostlocate informational

# Engineers Checklist

This PR introduces a new navbar and a landing page to the informational website. The navbar provides easy navigation across the website, while the landing page serves as the main entry point, presenting key information and guiding users to relevant sections of the site.



Motivation and Context: 

Adding a navbar improves the site's usability by offering a consistent navigation structure, making it easier for users to access different sections of the website. The landing page provides a professional and welcoming introduction to the website's content, enhancing user engagement from the moment they arrive.

Dependencies:

    Tailwild
    Next JS
    Figma design

What this PR does?

    What is this?
        This PR adds a responsive navbar and a visually appealing landing page to the informational website.

    Why this?
        To improve user experience by providing easy navigation and an engaging introduction to the website.

    How are you doing this?
        Implementing a responsive navbar component with links to the main sections of the website.
        Designing and integrating a landing page with key information and calls-to-action.


Type of Change

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)
Breaking change (fix or feature that would cause existing functionality to not work as expected)
This change requires a documentation update
Optimization
Improvement
Bug fix

How can it be tested?

    Navigate to the Website:
        Open the website in a browser.
Verify Navbar:Check if the navbar appears on all pages and includes links to different sections.
        Test responsiveness by resizing the browser window.
Verify Landing Page:
Ensure the landing page is displayed as the homepage.
        Verify that all visual elements and text are correctly styled and aligned.
    Check Navigation:
        Click on the links in the navbar to ensure they direct to the correct pages.

Screenshots:
![lostlocate_informational](https://github.com/user-attachments/assets/5390efd8-350c-4647-b90a-8d58591b438e)


Where is it documented?
https://wambuinatalygakenye.atlassian.net/browse/UP-101
